index.js

```diff
@@ -33,7 +33,7 @@ class LokiTransport extends Transport {
     })
 
     this.useCustomFormat = options.format !== undefined
-    this.labels = options.labels
+    this.labels = options.labels || { job: 'winston-loki' }
```

Comment:
This way of assignment doesn't make sense with a check below `if (this.labels)` because current implementation forbids `this.labels` to have falsy value. We should do
```js
this.labels = 'labels' in options ? options.labels : { job: 'winston-loki' }
```
or remove the `if (this.labels) {` line

I think the first option is more expectable